stages/org.osbuild.greenboot

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/python3
+"""
+Configure greenboot
+
+Update configuration of greenboot in /etc/greenboot/greenbot.conf.
+"""
+
+import sys
+
+import osbuild.api
+
+
+SCHEMA_2 = r"""
+"inputs": {
+  "type": "object",
+  "additionalProperties": false,
+  "properties": {
+    "monitor_services": {
+        "type": "array",
+        "items": {
+          "type": "string"
+        }
+    }
+  }
+},
+"options": {
+  "additionalProperties": false
+}
+"""
+
+
+def main(inputs, tree):
+    config_file = f"{tree}/etc/greenboot/greenboot.conf"
+
+    with open(config_file, "a", encoding="utf-8") as f:
+        f.write("\n")
+        f.write('GREENBOOT_MONITOR_SERVICES="{0}"'.format(" ".join(inputs["monitor_services"])))
```

Comment:
I am a tiny bit worried that this is brittle since it just appends values and does not replace them if they are found. We have other stages that take care of this with similar syntax, like https://github.com/osbuild/osbuild/blob/main/stages/org.osbuild.pwquality.conf

Reply:
alrighty, done, thanks for the pointer Christian!